example/examplejob.yaml

```diff
@@ -3,17 +3,17 @@ kind: TrainingJob
 metadata:
   name: example
 spec:
-  image: "paddlepaddle/paddlecloud-job"
+  image: "yancey1989/edl-example"
```

Comment:
image name?

Reply:
Done, changed to the official image name.